### PR TITLE
Adds 15 sites to DNS, realphabetizes some of the list

### PR DESCRIPTION
**Pull request by wslack**
Big update here, helping a bunch of new Federalist sites deploy

PRs affecting cloud.gov.tf or a Federalist site must receive approval from a member of the relevant team.

@fureigh to review, @datn to also approve before I merge

### CODE REVIEW

**Comment by datn** [comment]
Did you specifically want `autoapi` and `charlie`, which point at CloudFront distros, to stay CNAMEs?

Is `fec-style.18f.gov.18f.gov` an intended repetition of the domain name, or should that be `fec-style.18f.gov`? (I note both currenty resolve, and to the same thing)

**Comment by wslack** [comment]
> Did you specifically want autoapi and charlie, which point at CloudFront distros, to stay CNAMEs?

I don't manage those and didn't touch them except to reorder for alphabetization's sake.

> Is fec-style.18f.gov.18f.gov an intended repetition of the domain name, or should that be fec-style.18f.gov? (I note both currently resolve, and to the same thing)

Oh no, that's a mistake by me. Will fix.

**Comment by wslack** [comment]
@fureigh @datn any other comments? Would like to merge this ASAP given the pressures on cloud.gov to migrate.

**Comment by wslack** [comment]
@fureigh should it be 
`ttl = 300`

File: terraform/18f.gov.tf [hunk 1]
```diff
@@ -249,7 +307,25 @@ resource "aws_route53_record" "18f_gov_continua11y_18f_gov_cname" {
...
+    zone_id = "Z2FDTNDATAQYW2"
+    evaluate_target_health = false
+  }
+}
+
+resource "aws_route53_record" "18f_gov_dnssec_18f_gov_ns" {
+  zone_id = "${aws_route53_zone.18f_gov_zone.zone_id}"
+  name = "dnssec.18f.gov."
+  type = "NS"
+  ttl = 300
```
Comment (fureigh) [inline terraform/18f.gov.tf]: @wslack Here's an example.

Comment (wslack) [inline terraform/18f.gov.tf]: right, I meant do we want five minutes, 10 minutes, etc etc. I kept five min / 300.

Comment (fureigh) [inline terraform/18f.gov.tf]: That's good for quicker uptake. Once the migrations are all complete, let's come back and revise these to be longer, but that's outside the scope of this pull request. (cc @jcscottiii/@datn)

File: terraform/18f.gov.tf [hunk 2]
```diff
@@ -232,15 +295,18 @@ resource "aws_route53_record" "18f_gov_compliance-viewer_18f_gov_cname" {
   records = ["d1ij48sfge1oxq.cloudfront.net"]
 }
 
-resource "aws_route53_record" "18f_gov_dnssec_18f_gov_ns" {
+resource "aws_route53_record" "18f_gov_content-guide_18f_gov_a" {
```
Comment (fureigh) [inline terraform/18f.gov.tf]: @wslack I believe this hyphen needs to be an underscore.

File: terraform/18f.gov.tf [hunk 3]
```diff
@@ -267,6 +351,18 @@ resource "aws_route53_record" "18f_gov_dolores_staging_18f_gov_cname" {
   records = ["d9d5ti9w8z48u.cloudfront.net"]
 }
 
+resource "aws_route53_record" "18f_gov_fec-style_18f_gov_a" {
```
Comment (fureigh) [inline terraform/18f.gov.tf]: @wslack Hyphen patrol

File: terraform/18f.gov.tf [hunk 4]
```diff
@@ -633,6 +779,30 @@ resource "aws_route53_record" "18f_gov_pages_18f_gov_cname" {
...
+  type = "A"
+  ttl = 300
+  alias {
+    name = "dqd8t6xkgtofc.cloudfront.net."
+    zone_id = "Z2FDTNDATAQYW2"
+    evaluate_target_health = false
+  }
+}
+
+resource "aws_route53_record" "18f_gov_plain-language-tutorial_18f_gov_a" {
```
Comment (fureigh) [inline terraform/18f.gov.tf]: +1

File: terraform/18f.gov.tf [hunk 5]
```diff
@@ -684,10 +854,23 @@ resource "aws_route53_record" "18f_gov_tock_18f_gov_cname" {
   records = ["dualstack.production-star-18f-gov-elb-1963420885.us-gov-west-1.elb.amazonaws.com"]
 }
 
+resource "aws_route53_record" "18f_gov_writing-lab-guide_18f_gov_a" {
```
Comment (fureigh) [inline terraform/18f.gov.tf]: Another hyphen-to-underscore conversion waiting to happen.

**Comment by wslack** [comment]
@fureigh thanks! I think the hyphens are fine per https://federalist-modern-team-template.18f.gov/ and https://github.com/18F/dns/blob/deploy/terraform/18f.gov.tf#L305. Will fix accessibility now.

**Comment by fureigh** [comment]
@wslack I think you're right — thanks for the link! Just add a TTL parameter to that accessibility record, then I say :shipit:.

**Comment by wslack** [comment]
@fureigh done! Now waiting for a ✅ in this PR.

File: terraform/18f.gov.tf (hunk 1 above)
Comment (datn) [inline terraform/18f.gov.tf]: You're never going to see me lobbying for a longer TTL.

Comment (fureigh) [inline terraform/18f.gov.tf]: Ha. I suppose AWS can probably handle the frequent lookups.

**Pull request merged**